components/service/glean/build.gradle

```diff
@@ -15,7 +15,7 @@ apply plugin: 'kotlin-android'
  * This uses a specific version of the schema identified by a git commit hash.
  */
 String GLEAN_PING_SCHEMA_GIT_HASH = "1933f26"
-String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
+String GLEAN_PING_SCHEMA_URL = "https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/5489597276d168a7f275012ff2a83bcb51e233c2/schemas/glean/baseline/baseline.1.schema.json"
```

Comment:
This will need to change once mozilla-services/mozilla-pipeline-schemas#297 is merged.